src/index.ts

```diff
@@ -122,7 +122,7 @@ export {UpdateEvent} from "./subscriber/event/UpdateEvent";
 export {RemoveEvent} from "./subscriber/event/RemoveEvent";
 export {EntitySubscriberInterface} from "./subscriber/EntitySubscriberInterface";
 export {BaseEntity} from "./repository/BaseEntity";
-export {EntitySchema} from "./entity-schema/EntitySchema";
+export {EntitySchema, EntitySchemaTable, EntitySchemaColumn, EntitySchemaIndex, EntitySchemaRelation} from "./entity-schema/EntitySchema";
```

Comment:
Can you extract each interface into its own file? Looks like a mess there

Reply:
Ok will do ASAP